packages/features/ee/organizations/lib/onboardingStore.ts

```diff
@@ -151,22 +279,14 @@ export const useOnboarding = (params?: { step?: "start" | "status" | null }) =>
           slug: organizationOnboarding.slug,
           bio: organizationOnboarding.bio,
           logo: organizationOnboarding.logo,
+          brandColor: organizationOnboarding.brandColor,
+          bannerUrl: organizationOnboarding.bannerUrl,
         });
-        if (isAdmin && organizationOnboarding?.orgOwnerEmail !== session.data?.user.email) {
-          reset();
-        }
-      }
-    } else {
-      // First step doesn't require onboardingId
-      const requireOnboardingId = step !== "start";
-
-      // Reset to first step if onboardingId isn't available
-      if (!onboardingId && requireOnboardingId) {
-        console.warn("No onboardingId found in store, redirecting to /settings/organizations/new");
-        router.push("/settings/organizations/new");
       }
     }
-  }, [organizationOnboarding, isLoadingOrgOnboarding, isAdmin, onboardingId, reset, step, router]);
+    // Note: We no longer redirect if onboardingId is missing, as regular users
+    // don't create the onboarding record until the final step
+  }, [organizationOnboarding, isLoadingOrgOnboarding, isAdmin, reset, step, router, path]);
```

Comment:
_step_ variable isn't used now, so could be removed from this deps list and also from _useOnboarding_ argument I believe